scripts/docker-entrypoint.sh

```diff
@@ -101,8 +101,14 @@ if [[ "$1" != "/"* ]]; then
 
     if [[ "$1" = "gunicorn" ]]; then
         shift
+        EXTRA_PARAMS=""
+        SSL_KEY_PATH="/usr/src/paperless/data/ssl.key"
+        SSL_CERT_PATH="/usr/src/paperless/data/ssl.cert"
+        if [ "${PAPERLESS_USE_SSL}" = "true" ] && [ -f "${SSL_KEY_PATH}" ] && [ -f "${SSL_CERT_PATH}" ]; then
+            EXTRA_PARAMS="--certfile=${SSL_CERT_PATH} --keyfile=${SSL_KEY_PATH}"
+        fi
```

Comment:
We might wanna change this to print some error message, if the files are not found.
```sh
if [ "${PAPERLESS_USE_SSL}" = "true" ]; then
    if [ -f "${SSL_KEY_PATH}" ] && [ -f "${SSL_CERT_PATH}" ]; then
        EXTRA_PARAMS="--certfile=${SSL_CERT_PATH} --keyfile=${SSL_KEY_PATH}"
    else
        echo "Error: Could not find certfile in ${SSL_CERT_PATH} or keyfile in ${SSL_KEY_PATH}, but \$PAPERLESS_USE_SSL is true. Starting without SSL enabled.
    fi
fi
```

Reply:
I have kept the definitions of `$SSL_KEY_PATH` and `$SSL_CERT_PATH`, as it makes the setup easier. Or should those be configurable, as well?

Reply:
I'd leave that as is until someone comes along and needs it changed.